istio-proxy/istio-proxy.spec

```diff
@@ -137,42 +136,4 @@ TEST_ENVOY=false RUN_TESTS=true %{SOURCE2}
 * Tue Feb 11 2020 Brian Avery <bavery@redhat.com> - 1.1.0-1
 - Updated to Istio 1.4 proxy
 - Added support for building based on SHA
-- Simplified repositories
-* Mon Jan 13 2020 Kevin Conner <kconner@redhat.com> - 1.0.4-1
-- Bump version to 1.0.4
-
-* Thu Oct 17 2019 Jonh Wendell <jonh.wendell@redhat.com> - 1.0.2-1
-- Updated to Maistra 1.0.2, Istio-Proxy 1.1.17
-
-* Thu Jul 18 2019 William DeCoste <wdecoste@redhat.com>
-  Release 1.0.0-0
-* Mon Jul 15 2019 Brian Avery <bavery@redhat.com>
-- Update to Maistra 0.12 release
-* Thu Jun 20 2019 William DeCoste <wdecoste@redhat.com>
-  Release 0.12.0-1
-* Tue Jun 11 2019 William DeCoste <wdecoste@redhat.com>
-  Release 0.12.0-0
-* Tue May 14 2019 William DeCoste <wdecoste@redhat.com>
-  Release 0.11.0-0
-* Thu Mar 07 2019 Dmitri Dolguikh <ddolguik@redhat.com>
-  Release 0.9.0-2
-* Mon Mar 04 2019 Dmitri Dolguikh <ddolguik@redhat.com>
-  Release 0.9.0-1
-* Thu Feb 14 2019 Kevin Conner <kconner@redhat.com>
-  Release 0.8.0-1
-* Sun Jan 20 2019 Kevin Conner <kconner@redhat.com>
-  Release 0.7.0-1
-* Thu Dec 20 2018 Kevin Conner <kconner@redhat.com>
-  Release 0.6.0-1
-* Wed Nov 21 2018 Dmitri Dolguikh <ddolguik@redhat.com>
-  Release 0.5.0-1
-* Mon Oct 29 2018 Dmitri Dolguikh <ddolguik@redhat.com>
-  Release 0.4.0-1
-* Fri Oct 12 2018 Dmitri Dolguikh <ddolguik@redhat.com>
-  Release 0.3.0-1
-* Wed Sep 12 2018 Dmitri Dolguikh <ddolguik@redhat.com>
-  Release 0.2.0-1
-* Tue Jul 31 2018 Dmitri Dolguikh <ddolguik@redhat.com>
-- Release 0.1.0-1
-* Mon Mar 5 2018 Bill DeCoste <wdecoste@redhat.com>
-- First package
+- Simplified repositories
```

Comment:
Why removing this?

Reply:
This is a change pulled over from product. I think that it's worth a discussion. This is a new release moving in its own direction. Should it extend the same changelog?

Reply:
I believe we should start a new change log.  I think "Updated to Istio 1.4" says enough, and most likely includes everything that was in the previous change log.

Reply:
I disagree. Changelogs in .spec should persist. See for instance the changelog for bash: https://src.fedoraproject.org/rpms/bash/blob/f32/f/bash.spec